### PR TITLE
Announcement Blog for '22 GC Elections

### PR DESCRIPTION
**Pull request by austinlparker**

### CODE REVIEW

File: content/en/blog/2022/gc-election-results.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+election saw 169 ballots cast with 386 eligible voters. While our participation
+percentage was down slightly, we're thrilled that we've welcomed so many new
+project members over the past year!
+
+With that out of the way, please joining me in congratulating [Aloita
+Sharma](https://github.com/alolita), [Daniel Dyla](https://github.com/dyladan),
+and [Morgan McLean](https://github.com/mtwo) on their re-election! They will be
+joined by long-time contributor and maintainer of OpenTelemetry Java
+Instrumentation [Trask Stalnaker](https://github.com/trask)! They will all serve
+a two-year term.
```
Comment (cartermp) [inline content/en/blog/2022/gc-election-results.md:23]:
```suggestion
Instrumentation [Trask Stalnaker](https://github.com/trask)! All will serve
a two-year term.
```
Probably a little cleaner to not have two different "they"s.

**Review by cartermp — APPROVED** [review]

File: content/en/blog/2022/gc-election-results.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+OpenTelemetry Governance Committee (GC) election! Before we get into the
+results, I'd like to share some statistics about the election!
+
+This year, 541 project members were eligible to vote. Of those, 219 cast
+ballots, for a participation rate of nearly 41%! For comparison, the 2021
+election saw 169 ballots cast with 386 eligible voters. While our participation
+percentage was down slightly, we're thrilled that we've welcomed so many new
+project members over the past year!
+
+With that out of the way, please joining me in congratulating [Aloita
```
[the remark this answers is not in the document]
Comment (jpkrohling) [reply content/en/blog/2022/gc-election-results.md:18]:
```suggestion
With that out of the way, please join me in congratulating [Alolita
```

File: content/en/blog/2022/gc-election-results.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+OpenTelemetry Governance Committee (GC) election! Before we get into the
+results, I'd like to share some statistics about the election!
+
+This year, 513 project members were eligible to vote. Of those, 219 cast
+ballots, for a participation rate of nearly 43%! For comparison, the 2021
+election saw 169 ballots cast with 386 eligible voters. Participation was 
+roughly flat year-over-year, and we're thrilled that we've welcomed so 
+many new project members and maintained a high participation rate.
+
+With that out of the way, please joining me in congratulating [Alolita
```
Comment (jpkrohling) [inline content/en/blog/2022/gc-election-results.md:18]: Can you confirm you did mean "joining" here? As a non-native speaker, I would write "join me in" instead, but I'm probably wrong.

Comment (austinlparker) [reply content/en/blog/2022/gc-election-results.md]: I did not! Good catch

**Review by alolita — APPROVED** [review]
+1

**Comment by cartermp** [comment]
Will merge now as there are no other approved blogs in the queue. We can then promote on whatever schedule we like.

**Pull request merged**